openreview/arr/management/setup_shared_data.py

```diff
@@ -91,6 +91,15 @@ def _is_not_available(month: str, year: int, current_date: datetime.datetime, po
             f"{previous_cycle_id}/{domain.content['ethics_reviewers_name']['value']}",
         ]
     ]
+    all_profiles = openreview.tools.get_profiles(client, client.get_group(domain.content['reviewers_id']['value']).members)
```

Comment:
a better option is to merge all the members first and call to tools.get_profiles only once.